src/main/resources/trellofunctionality/trellologin.story

```diff
@@ -13,20 +13,18 @@ When I click on element located `//button[@data-testid='header-member-menu-logou
 When I click on element located `//button[@id='logout-submit']`
 And I wait until element located `//*[@id="BXP-APP"]/header[2]/div[1]/div/a` appears
 
-Examples: 
+Examples:
 |userid|paasword|
 |epamnikita@gmail.com|epamnikita@2020|
 
-Scenario: login to trello home page with invalid credentials 
+Scenario: login to trello home page with invalid credentials
 Given I am on a page with the URL 'https://trello.com/home'
 When I type `<userid>` in field located `//input[@id='user']`
 And I click on element located `//input[@id='login']`
 When I type `<password>` in field located `//input[@id='password']`
 And I click on element located `//button[@id='login-submit']`
 And I wait until element located `//div[@id="login-error"]` appears
 
-Examples: 
+Examples:
```

Comment:
There's no need to use example table if there's only one set of data, i.e. scenario is not iterated